crates/bevy_ecs/src/lib.rs

```diff
@@ -34,9 +34,10 @@ pub mod prelude {
         event::{EventReader, EventWriter, Events},
         query::{Added, AnyOf, ChangeTrackers, Changed, Or, QueryState, With, Without},
         schedule::{
-            AmbiguitySetLabel, ExclusiveSystemDescriptorCoercion, ParallelSystemDescriptorCoercion,
-            RunCriteria, RunCriteriaDescriptorCoercion, RunCriteriaLabel, Schedule, Stage,
-            StageLabel, State, SystemLabel, SystemSet, SystemStage,
+            AmbiguitySetLabel, AmbiguitySetLabelId, ExclusiveSystemDescriptorCoercion,
```

Comment:
I don't think we want to add the new types to the prelude? They really don't seem useful enough.

Reply:
Seems reasonable, it's easier to add them later if we end up wanting to.